README.md

```diff
@@ -45,7 +45,72 @@ Run the following commands from the root of the project directory:
 
 Open your forked repo in your browser and then raise a PR to the `master` branch of this repository!
 
+## Usage
+Once the executable file is created, the program can be executed as:
 
+### Without Flag Options
+This will by default show all the parameters of the System. It will provide all the parameters of CPU, Memory and Network Status.
+```console
+./termiboard
+```
+Output:
+```console
+ _                      _ _                         _
+| |_ ___ _ __ _ __ ___ (_) |__   ___   __ _ _ __ __| |
+| __/ _ \ '__| '_ ` _ \| | '_ \ / _ \ / _` | '__/ _` |
+| ||  __/ |  | | | | | | | |_) | (_) | (_| | | | (_| |
+ \__\___|_|  |_| |_| |_|_|_.__/ \___/ \__,_|_|  \__,_|
+v1.0
+CPU Model: Intel(R) Core(TM) i5-7200U CPU @ 2.50GHz
+CPU Cores: 4
+CPU Usage: 12.50%
+Ram Used: 82.000000%
+Ram Available: 1.39 GB
+Disk Usage: 66.69%
+Disk Space Available: 277.50 GB
+Local IP Address:
+        [Ethernet] 
+                IPv6 -> 
+                IPv4 -> 
+        [Local Area Connection* 2] 
+                IPv6 -> 
+                IPv4 -> 
+        [WiFi] 
+                IPv6 -> 
+                IPv4 -> 
+        [Loopback Pseudo-Interface 1]
+                IPv6 -> ::1/128
+                IPv4 -> 127.0.0.1/8
+Public IPv4 Address: 
+```
```

Comment:
Maybe we can a small screenshot rather than a console output, we have coloured outputs

Reply:
Will it be okay to create a Screenshot folder to store the images/screenshots required? Like Assets?